tests/storage/storage.go

```diff
@@ -1098,7 +1111,12 @@ var _ = SIGDescribe("Storage", func() {
 				Expect(disks[0].BackingStore.Source.Dev).To(Equal(converter.GetBlockDeviceVolumePath("disk0")))
 			})
 			It("should generate the pod with the volumeDevice", func() {
-				vmi = newVMIWithEphemeralPVC(dataVolume.Name)
+				vmi = libvmi.New(
+					libvmi.WithNamespace(testsuite.GetTestNamespace(nil)),
+					libvmi.WithResourceMemory("256Mi"),
```

Comment:
how about using `NewGuestless` ? I don't think that the test needs anything to run inside the VM.